test/e2e/performanceprofile/functests/6_mustgather_testing/mustgather.go

```diff
@@ -101,7 +101,7 @@ var _ = Describe("[rfe_id: 50649] Performance Addon Operator Must Gather", func(
 			for _, item := range mcps.Items {
 				mcpFiles = append(mcpFiles, fmt.Sprintf("cluster-scoped-resources/machineconfiguration.openshift.io/machineconfigpools/%s.yaml", item.Name))
 			}
-			err := checkfilesExist(mcpFiles)
+			err = checkfilesExist(mcpFiles)
```

Comment:
this commit should be squashed in the one with the original error

Reply:
done